src/ObjectIndexer.php

```diff
@@ -55,4 +59,19 @@ public function removeById(string $objectId, $type)
 
         $this->index->delete($params);
     }
+
+    private function getIndexParams(Indexable $object, string $type = "_doc")
+    {
+        if (!$object->shouldBeIndexed()) {
+            return;
```

Comment:
On a pure semantic point of view, I wonder if returning `null` isn't better than returning `void`, given that the this is a getter function so "something" is expected, event if it is a null value. (yes, this is clearly nitpicking)